components/count-badge/README.md

```diff
@@ -0,0 +1,28 @@
+# Count Badge
+
+## d2l-count-badge
+
+The `d2l-count-badge` element is a web component to display a number count, either in a `"notification"` or `"count"` style.
+
+
+![Notification Badge](./screenshots/count-badge-notification-small.png?raw=true)
+![Count Badge](./screenshots/count-badge-count-large.png?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/count-badge/count-badge.js';
+</script>
+<d2l-count-badge badge-size="small" badge-type="notification" number="100"></d2l-count-badge>
+```
+
+**Properties:**
+
+| Property | Type | Description |
+|--|--|--|
+| `number` | Number, required | The number to display on the badge. |
+| `badge-size`, default: `small` | String | The size of the badge. Valid options are `"small"` and `"large"`. |
+| `badge-type`, default: `count` | String | The type of the badge. Valid options are `"notification"` and `"count"`. Notification badges are orange and truncate numbers above 99 to `99+`, while count badges are grey and do not truncate the number. |

```

Comment:
Nit: could probably go with just `size` and `type` as attribute names here, since we know it's for the badge.